docs/subsystems/hotspots.md

```diff
@@ -30,7 +30,7 @@ ALL_HOTSPOTS = {
 ### Step 2: Configure hotspot placement
 
 The target element and visual orientation of each hotspot is specified in
-`HOTSPOT_LOCATIONS` of `web/src/hotspots.js`.
+`HOTSPOT_LOCATIONS` of `web/src/hotspots.ts`.
```

Comment:
You can just include this in the main commit; it's logically part of the typescript conversion change, since that renames the file.